content/raw/general/onboarding.md

```diff
@@ -0,0 +1,260 @@
+- If the seed phrase suggestions is displayed and the user presses the DOWN key, it must select the next suggestion in the list
+- If the seed phrase suggestions is displayed and the user presses the UP key, it must select the previous suggestion in the list
+- The submit button MUST NOT be enabled until each and every input contain valid seed phrase words
+- If the user submits the seed phrase, and the seed phrase is invalid or fails a checksum then the error message "Invalid seed" MUST BE displayed
+- If the user submits the seed phrase, and the seed phrase is invalid or fails a checksum then the submit button MUST NOT be enabled until at least one of the input changes
+- If the user submits the seed phrase and it's valid then the "account syncing screen" MUST BE displayed
+
+#### Existing User Keycard Login Import screen
+
+- The keycard logo MUST BE shown
+- If no reader is detected the message "Plug in Keycard reader.." MUST be displayed
+- If a reader is detected but no keycard is present the message "Insert your Keycard" MUST be displayed
+- If a keycard is present then the loading message "Reading Keycard..." MUST be displayed
+- If the keycard does not store a profile, then the error message "This keycard stores keys but doesn’t store a profile" MUST BE displayed
+- If the keycard does not store a profile, then options "Factory reset" and "Cancel" must be displayed
+- TODO
+- If the keycard is empty then the error message "This Keycard is empty" MUST BE displayed
+- If the keycard is empty then the option "Generate keys for a new Keycard" MUST BE displayed
+    - TODO
+- If the keycard is locked with a PIN then a "Enter Keycard PIN" message MUST BE displayed
+- If the keycard is locked with a PIN then a 6 Digit input MUST BE displayed
+- If the keycard pin is incorrect then the error message "PIN incorrect" MUST be displayed
+- The maximum number of possible PIN attempts MUST BE 3
+- If the keycard pin is incorrect then the number of possible attempts MUST decrease by 1 and the message "X attempts" remaining MUST BE displayed
+- If the total number of failed attemps reaches the maximum number then the keycard logo MUST BE reaplced with a logo indicating an error
+- If the total number of failed attemps reaches the maximum number then the error message "Keycard locked" MUST BE displayed
+- If the total number of failed attemps reaches the maximum number then the option "Unlock your Keycard" MUST BE displayed
+    - TODO https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=7971%3A412047
+- If the keycard login is successful then a Biometrics screen must be displayed
+- The Biometrics screen MUST display a biometrics logo
+- The Biometrics screen MUST display the options "Yes, use Touch ID" and "I prefer to use my PIN"
+- Choosing the option "Yes, use Touch ID" MUST display the biometrics screen
+    - TODO: unclear here what should happen
```

Comment:
@iurimatias in this step we should store the pin to the keychain and use it with the next app run to login automatically a user when reading the pin from the keychain.